core/src/components/range/range.tsx

```diff
@@ -861,8 +767,7 @@ Developers can dismiss this warning by removing their usage of the "legacy" prop
   }
 
   render() {
-    const { legacyFormController } = this;
-    return legacyFormController.hasLegacyControl() ? this.renderLegacyRange() : this.renderRange();
+    return this.renderRange();
```

Comment:
Can we just move the `renderRange` function into `render` instead?

Reply:
Fixed in https://github.com/ionic-team/ionic-framework/pull/29040/commits/a65ec85a4d04a6ac46f66633f86ce792dead1e05